pages/common/gnmic.md

```diff
@@ -6,20 +6,20 @@
 
 - Request device capabilities:
 
-`gnmic --address {{ip:port}} --insecure capabilities`
+`gnmic --address {{ip:port}} capabilities`
 
 - Provide a username and password to fetch device capabilities:
 
-`gnmic --address {{ip:port}} --username {{username}} --password {{password}} --insecure capabilities`
+`gnmic --address {{ip:port}} --username {{username}} --password {{password}} capabilities`
```

Comment:
```suggestion
`gnmic --address {{ip:port}} --username {{value}} --password {{value}} capabilities`
```

It's better not to repeat when possible.

Reply:
Sounds good

Reply:
There is formal rule for this now but it should be. At the end of the day I can mark this conversation as resolved due to the reason mentioned earlier.

Reply:
When has this been added as a rule?

We should always use informative placeholders

Reply:
I've missed "no".